jsk_fetch_robot/jsk_fetch_startup/launch/fetch_bringup.launch

```diff
@@ -151,6 +152,16 @@
     <arg name="hostname" value="$(arg hostname)" />
   </include>
 
+  <!-- google chat -->
+  <include if="$(arg launch_google_chat)"
+           file="$(find google_chat_ros)/launch/google_chat.launch">
+    <arg name="use_yaml" value="true" />
+    <arg name="recieving_mode" value="url" />
```

Comment:
I assume that `recieving` is a typo, but it may be OK not to change original node. https://github.com/knorth55/jsk_3rdparty/blob/2daa09ca6a00d3b6c16239f0e41edf36e22c42f2/google_chat_ros/launch/google_chat.launch#L2

Reply:
Sorry,,,,